common/src/main/java/com/wynntils/handlers/chat/ChatHandler.java

```diff
@@ -451,6 +440,6 @@ private RecipientType getRecipientType(StyledText codedMessage, MessageType mess
     }
 
     private boolean shouldSeparateNPC() {
```

Comment:
You might consider inlining this at this point...

Reply:
I think it can stay until we refactor NPC logic out of there.